test/magic.test.js

```diff
@@ -1165,6 +1166,147 @@ describe('magic tests', () => {
           });
         });
       });
+
+      describe('stream encryption/decryption', () => {
+        before(() => {
+          this.HEADER_BYTES = sodium.crypto_secretstream_xchacha20poly1305_HEADERBYTES + 1;
+        })
+
+        beforeEach(() => {
+          readStream = fs.createReadStream('./test/plaintext.txt');
+          writeStream = fs.createWriteStream('./test/decryptedtext.txt');
```

Comment:
Do we have a clean up for this file at the end of the tests?

Reply:
Yep, You're right. I'm adding an `after` function.